UIMod/onboard_bundled/twoboxform/twoboxform.js

```diff
@@ -138,6 +138,12 @@ document.addEventListener('DOMContentLoaded', () => {
                 
             } else if (configField === "SaveInfo") {
                 const primaryValue = document.getElementById('primary-field').value.trim();
+                // If the world name contains a space, it's invalid 
+                if (primaryValue.includes(' ')) {
+                    showNotification('The world name cannot contain spaces!', 'error');
```

Comment:
btw, users will fight us on this, since the game **absolutely** allows one to use world names with spaces in them: 

<img width="1086" height="104" alt="image" src="https://github.com/user-attachments/assets/8f767e37-b989-4443-84a3-7ab217f352f3" />

<img width="521" height="27" alt="image" src="https://github.com/user-attachments/assets/b1a6ee0f-2fae-4834-9cab-12c45b261e43" />